LISA/Downloader.cpp

```diff
@@ -181,11 +186,23 @@ int Downloader::curlProgressCb(void* userData,
 
 bool Downloader::onProgress(long dlTotal, long dlNow)
 {
-    TRACE_L1("downloaded: %ld/%ld", dlNow, dlTotal);
-    // TODO progress notification
+    Progress newProgres = {dlTotal, dlNow};
+    if (newProgres != progress) {
+        progress = newProgres;
```

Comment:
This assignment should probably by synchronized - LockGuard lock(taskMutex);

Reply:
synchronization is done in Executor.cpp in setProgress